packages/salesforcedx-vscode-core/src/commands/forceAuthWebLogin.ts

```diff
@@ -102,8 +108,31 @@ export class ForceAuthWebLoginDemoModeExecutor extends ForceAuthDemoModeExecutor
   }
 }
 
-export class AliasGatherer implements ParametersGatherer<Alias> {
-  public async gather(): Promise<CancelResponse | ContinueResponse<Alias>> {
+export class AuthParamsGatherer implements ParametersGatherer<AuthParams> {
+  public async gather(): Promise<
+    CancelResponse | ContinueResponse<AuthParams>
+  > {
+    const orgTypeItems: vscode.QuickPickItem[] = [
+      { label: 'Production', detail: 'login.salesforce.com' },
```

Comment:
These should also be labels in the `i18n` file since this is the text that we show the user while running the command. cc @ruthemmanuelle 

![screen shot 2018-12-14 at 2 08 06 pm](https://user-images.githubusercontent.com/1041842/50032220-ba013f80-ffa9-11e8-8ca1-4898cc786964.png)

Reply:
Does this tie into the sfdx-project.json in any way? By that I mean do you read that file first to see what the user has put into the `sfdxLoginUrl` field and make that the default option in the prompt?

It seems that it should at least read what you have provided in the sfdx-project.json (including any custom urls) and not need to override it each time.

What I fear is that the user has already coded some url in sfdx-project.json and now in this flow, you are forcing them to have to type it into the custom field again.

Thoughts?

Reply:
I think the sfdx-core module might provide facilities for parsing the sfdx-project.json to help with this use case.

@allileong - please keep me honest here.

Reply:
Yes, this code has not yet been submitted, but it shows an example of how to read properties from the `sfdx-project.json` using the sfdx-core module.

Reply:
Cool, I think that's a good idea to surface the value in `sfdx-project.json` as the placeholder when you select "Custom"

Reply:
<img width="602" alt="screen shot 2018-12-17 at 2 00 14 pm" src="https://user-images.githubusercontent.com/10779873/50118234-eb754780-0203-11e9-8a53-f0b960772b80.png">
<img width="601" alt="screen shot 2018-12-17 at 2 07 29 pm" src="https://user-images.githubusercontent.com/10779873/50118579-ea90e580-0204-11e9-9707-8a66fe4ba8f6.png">

How do we feel about something like this? @vazexqi

Reply:
@brpowell - This looks good to me. I think using the second screenshot might be better since it's more consistent with the positions of Production and Sandbox, i.e., the URL is on the second line.

I will leave the wording to @ruthemmanuelle.

Thanks for making this change!

Reply:
Looks good. Just change "url" to "URL".